examples/io/plot_read_events.py

```diff
@@ -0,0 +1,49 @@
+"""
+.. _ex_read_events:
+
+=====================
+Reading an event file
+=====================
+
+Read events from a file. For a more detailed guide on how to read events
+using MNE-Python, see :ref:`tut_epoching_and_averaging`.
+"""
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Chris Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+
+# Reading events
+# events = mne.read_events(fname)  # all
```

Comment:
ok latest push does a full sphinx-gallery treatment and adds an explanation of what the columns are since this question keeps coming up